### ISSUE TITLE
Visual Studio 2019 - Package Manager Console: Cannot load Windows PowerShell snap-in

### ISSUE THREAD

**Issue by MV-QB** [PowerShell/PowerShell#15540]
Hello,

I´m using Microsoft Visual Studio 2019 Enterprise on Windows 10. 
Some days ago I started Visual Studio an got the following error in the Package Manager Console:

![image](https://user-images.githubusercontent.com/84380483/121141221-06f0c480-c83b-11eb-9fc7-8b26e27f05ec.png)
```
Cannot load Windows PowerShell snap-in Microsoft.PowerShell.Utility because of the following error: Could not load file or assembly 'Microsoft.PowerShell.Commands' or one of its dependencies. Das System kann die angegebene Datei nicht finden.
```
My VS Version is: Version 16.10.0

I already searched for solutions, repaired Visual Studio with the Visual Studio Installer, removed the NuGet-Package Manager and reinstalled it, but nothing worked.

## Environment data

```
Name                           Value
----                           -----
PSVersion                      5.1.19041.1023
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.19041.1023
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
```

**Comment by rjmholt** [comment]
This seems to be an issue with Windows PowerShell 5.1 interacting with a Visual Studio feature. This repository maintains PowerShell 7, which is now different from PS 5.1 in a few important ways. [You'll probably find good support from the PowerShell community](https://docs.microsoft.com/en-us/powershell/scripting/community/community-support).

**Comment by AlexShi23** [comment]
@MV-QB , if you have found a solution, please share it with me. I faced the same problem and could not find information on it.

**Comment by MV-QB** [comment]
@AlexShi23 I still don´t know where the error came from, but I solved the problem by downloading the MediaCreationTool from Microsoft and created an ISO file. After that I startet die Setup.exe from the ISO. I "Reinstalled" Windows 10 with the option to keep my apps and documents. At first place I had an error 0xC1900101 - 0x2000c. I disconnected my dockingstation and tryed it again. Now it run successfully and I can use PowerShell again.
https://stackoverflow.com/questions/67965269/windows-powershell-microsoft-powershell-commands-utility-not-found/67982788#67982788
